src/components/Markdown/Markdown.tsx

```diff
@@ -38,6 +38,12 @@ const handleDetails = (text: string): string | null => {
     return `<blockquote class="${style.details}">${result}</blockquote>`
 }
 
+const handleLogos = (text: string): string | null => {
+    const hasLogo = text.match(/\bLOGO:[a-z]+\b/);
+    if (!hasLogo) return null
+    return text.replace(/\bLOGO:([a-z]+)\b/g, `<img src="/images/logo-$1.png" class="${style.logo}" />`)
```

Comment:
@lens0021 can you move the logos to a new directory `logos/` where you would put all the logo images with names without the `logo-` prefix in the file names?

```suggestion
    return text.replace(/\bLOGO:([a-z]+)\b/g, `<img src="/images/logos/$1.png" class="${style.logo}" />`)
```

Reply:
Done. And I've removed the 41–42 lines.